Xamarin.Forms.Platform.iOS/Renderers/FrameRenderer.cs

```diff
@@ -25,7 +25,13 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 
 		void SetupLayer()
 		{
-			Layer.CornerRadius = 5;
+			float cornerRadius = Element.CornerRadius;
+
+			if (cornerRadius == -1f)
+				cornerRadius = 5f; // default corner radius
```

Comment:
It's not an issue in this particular case, but it's still a bad idea in general. Any equality comparison between floating point numbers should raise a red flag.